docs/data_node/extract_node/oracle-cdc.md

```diff
@@ -427,6 +500,36 @@ _Note: the mechanism of `scan.startup.mode` option relying on Debezium's `snapsh
 
 The Oracle Extract Node can't work in parallel reading, because there is only one task can receive change events.
 
+### Whole Database, Multiple Schemas, Multiple Tables Migration
```

Comment:
It is recommend to add a title for it such as '##Features'.

Reply:
Its upper level is a '## Features'